### PR TITLE
Apply fix for OCCI strings

### PR DESCRIPTION
**Pull request by ggovi**
Added pre-load trick

### CODE REVIEW

**Comment by ggovi** [comment]
please test

**Comment by smuzaffar** [comment]
@ggovi , no need to do this. I have updated SCRAM to support something like in test/BuildFile.xml
```
<ifcxx11_abi value="1">
  <flags SETENV="LD_PRELOAD=$(CMS_ORACLEOCCI_LIB)"/>
</ifcxx11_abi>
```
Once we have tonight's DEVEL IB then I will make PR for this.

File: CondTools/Ecal/test/EcalADCToGeV_update_test.sh [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #!/bin/sh
+export LD_PRELOAD=$CMS_ORACLEOCCI_LIB
```
Comment (fwyzard) [inline CondTools/Ecal/test/EcalADCToGeV_update_test.sh]: @smuzaffar would it make sense to make `cmsenv` do this automatically ?

**Comment by smuzaffar** [comment]
@fwyzard , we should not set it in env otherwise every executable is going to see it and any stdd::string::length call will go through it.

For unit tests, we are going to control it via BuildFile.xml e.g.
```
<ifcxx11_abi value="1">
  <flags SETENV="LD_PRELOAD=${CMS_ORACLEOCCI_LIB}"/>
</ifcxx11_abi>
```

**Comment by fwyzard** [comment]
Do you mean non-CMS(SW) executables ?
Ah, yes, you are right of course.

**Comment by ggovi** [comment]
superseded by #22366

**Pull request closed without merging**